lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1992,11 +1994,18 @@ def setUp(self):
         reload_django_url_config()
         super(TestRenderXBlock, self).setUp()
 
-    def get_response(self, url_encoded_params=None):
+    def test_render_xblock(self):
```

Comment:
@asadazam93, test name should be reflecting the specific case(s) that you are testing in there.